### ISSUE TITLE
3.5 Dev Proposals

### ISSUE THREAD

**Issue by unrelenting-d1x** [NEST-Protocol/nips#13]
A pull request contains several development proposals on the direction of NEST future development has been merged into [main](https://github.com/NEST-Protocol/nips) branch. 

We plan to release these features as NEST 3.5 (pending on the outcome of the discussion)

We hereby call for suggestions and ideas on these proposals from all NEST community members. And happy to hear any feedback on the scope of features, the varies of parameters, possible security implications and the design of the mechanisms

Looking forward to hearing the community!

**Comment by sunphen** [comment]
期待3.5版本上线
